share/wake/lib/core/json.wake

```diff
@@ -423,6 +423,7 @@ def mergeValueList (describeCompat: Boolean) (values: List JValue): Result JValu
             if eqFn x y then
                 Pass x
             else
+                # wake-format off
                 compatibilityMessage "N" "n" "ot every type can combine multiple different values. The incompatible values are: {map wrapFn values | formatExamples}"
```

Comment:
Sorry about this!

Reply:
I didn't know about that function but that's dedication to correct error messages and exactly the kind of stuff that makes i18n a giant fucking pain hahaha.

Reply:
You wouldn't know about it because I wrote that function specifically for that message...